lib/ansible/cli/galaxy.py

```diff
@@ -301,6 +302,10 @@ def add_list_options(self, parser, parents=None):
 
         list_parser.add_argument(galaxy_type, help=galaxy_type.capitalize(), nargs='?', metavar=galaxy_type)
 
+        if galaxy_type == 'collection':
+            list_parser.add_argument('-o', '--output', dest='output_format', choices=('human', 'yaml', 'json'), default='human',
```

Comment:
we use -o for output file elsewhere, it is not good to overload options with diff meanings

Reply:
Fair enough. How about `-f` / `--output-format`?

Reply:
Could use `--format` to be similar to pip. I wouldn't add `-f` though since it's already associated with `--force`.

Reply:
Right, missed that. How about `--output-as`?

Reply:
I think `--format` makes sense. Let's go with that. Thanks all.